rust/sbp2json/src/lib.rs

```diff
@@ -67,7 +67,9 @@ where
         sink.send_all(source)?;
     } else {
         while let Some(msg) = source.next() {
-            sink.send(msg?)?;
+            if let Ok(msg) = msg {
```

Comment:
We should probably write the error message to stderr (or log with `warn!` so the message can be enabled/disabled appropriately) instead of silently dropping.